src/Twisto/Invoice.php

```diff
@@ -147,12 +152,38 @@ public function returnAll()
         $this->deserialize($data);
     }
 
+
+    /**
+     * Split invoice to new one
+     * @param ItemSplit[] $items
+     * @return JSON response with new invoice
+     */
+    public function splitItems($items)
+    {
+        $data = array(
+            'items' => array_map(function(ItemSplit $item) {
+                return $item->serialize();
+            }, $items)
+        );
+
+        $data = $this->twisto->requestJson('POST', 'invoice/' . urlencode($this->invoice_id) . '/split/', $data);
+        $split_invoice = new Invoice($this->twisto, null);
+        $split_invoice->deserialize($data);
+        $this->get();
+
+        return $split_invoice;
+    }
+
+
     private function deserialize($data)
     {
         $this->invoice_id = $data['invoice_id'];
         $this->eshop_invoice_id = $data['eshop_invoice_id'];
         $this->customer_email = $data['customer_email'];
 
+        if ($data['parent_invoice_id'])
```

Comment:
Tohle můžeš přiřadit vždy. Buď tam bude ta hodnota vyplněná, nebo to bude `null`. Stejně jako `eshop_invoice_id`.